includes/forms/template.php

```diff
@@ -128,6 +141,14 @@ function give_get_donation_form( $args = array() ) {
                 <input type="hidden" name="give-form-minimum"
                        value="<?php echo give_format_amount( give_get_form_minimum_price( $form->ID ), array( 'sanitize' => false ) ); ?>"/>
 
+                <!-- The following field is for currency setting.-->
+                <input type="hidden" name="give-currency" value="<?php echo esc_attr( $form_currency ); ?>"/>
```

Comment:
We can use `esc_attr_e` so that we don't need to write `echo` as `esc_attr_e` will echo it by itself 

For reference: https://core.trac.wordpress.org/browser/tags/4.8/src/wp-includes/l10n.php#L259